code/processes/tickerlogreplay.q

```diff
@@ -198,18 +198,17 @@ replaylog:{[logfile]
 	@[`.;`upd;:;.replay.realupd]];
  .replay.tablecounts:.replay.errorcounts:.replay.pathlist:()!();
  .replay.replaydate:"D"$-10#string logfile;
+ if[ .replay.clean and (`$(string .replay.replaydate)) in key hdbdir;
```

Comment:
``(`$string .replay.replaydate)`` - can remove a set of parentheses

Reply:
Can also assign `string .replay.replaydate` to a variable here and refer to that in the next line, rather than re-evaluating